packages/suite/src/components/suite/FormattedCryptoAmount.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+import { HiddenPlaceholder, Sign } from '@suite-components';
+import { useSelector } from '@suite-hooks/useSelector';
+import { NetworkSymbol } from '@wallet-types';
+import { BitcoinUnits } from '@suite-reducers/suiteReducer';
+import { networkAmountToSatoshi } from '@wallet-utils/accountUtils';
+import { isValuePositive, SignValue } from '@suite-components/Sign';
+
+const Value = styled.span`
+    font-variant-numeric: tabular-nums;
+    overflow: hidden;
+    text-overflow: ellipsis;
+`;
+
+const Symbol = styled.span<{ isLowercase: boolean }>`
```

Comment:
```suggestion
const Symbol = styled.span<{ isLowerCase: boolean }>`
```
or `shouldBeUpperCased`

Reply:
done